downstream/modules/platform/proc-gs-auto-dev-create-automation-execution-proj.adoc

```diff
@@ -21,13 +21,13 @@ Procedure
 * Optional: *Description*
 * *Organization* (required): A project must have at least one organization. Select one organization now to create the project. When the project is created you can add additional organizations.
 * Optional: *Execution Environment*: Enter the name of the {ExecEnvShort} or search from a list of existing ones to run this project.
-For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_upgrade_and_migration_guide/upgrading-to-ees[Migrating to automation execution environments] in the _Red Hat Ansible Automation Platform Upgrade and Migration Guide_.
+For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_upgrade_and_migration_guide/upgrading-to-ees[Migrating to automation execution environments] in the Red Hat Ansible Automation Platform Upgrade and Migration Guide.
```

Comment:
Use attributes for both AAP and upgrade and migration guide.

Reply:
updated